server/db/models/reacto.js

```diff
@@ -0,0 +1,27 @@
+const Sequelize = require('sequelize')
+const db = require('../db')
+
+const Reacto = db.define('reacto', {
+  gistLink: {
+    type: Sequelize.STRING,
+    allowNull: false,
+    unique: true
+  },
+  replLink: {
+    type: Sequelize.STRING,
+    allowNull: false,
+    unique: true
+  },
+  fellowAssigned: {
```

Comment:
Instead of Fellow Assigned, it will be a One-To-Many Relationship between REACTO and User(Fellows). We don't need this field in the model. REACTO table will have a userId